lib/services/project-changes-service.ts

```diff
@@ -227,11 +230,9 @@ export class ProjectChangesService implements IProjectChangesService {
 
 			let fileStats = this.$fs.getFsStats(filePath);
 
-			let changed = fileStats.mtime.getTime() >= this._outputProjectMtime || fileStats.ctime.getTime() >= this._outputProjectCTime;
-			if (!changed) {
-				let lFileStats = this.$fs.getLsStats(filePath);
-				changed = lFileStats.mtime.getTime() >= this._outputProjectMtime || lFileStats.ctime.getTime() >= this._outputProjectCTime;
-			}
+			let changed =  fileStats.mtime.getTime() >= this._outputProjectMtime ||
```

Comment:
Why is a `greater than or equal` comparison made here? Wouldn't `changed` flip to true if the 2 sides were equal?